FxAClient/Frontend/SignIn/FxAGetStartedViewController.swift

```diff
@@ -0,0 +1,135 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+import WebKit
+
+protocol FxAGetStartedViewControllerDelegate {
+    func didStart() -> Void
+    func didCancel() -> Void
+}
+
+/**
+ * An interstitial controller that shows an icon, an optional error, and
+ * a button (replaced by a spinner) while another resource is loading.
+ */
+class FxAGetStartedViewController: UIViewController {
+    var delegate: FxAGetStartedViewControllerDelegate?
+
+    // Access this only on the main UI thread.
+    var waitingForReady: Bool = true
+
+    private var icon: UIImageView!
+    private var error: UILabel!
+    private var button: UIButton!
+    private var spinner: UIActivityIndicatorView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.Cancel, target: self, action: "didCancel")
+
+        view.backgroundColor = UIColor(red: 242/255.0, green: 242/255.0, blue:242/255.0, alpha:1.0)
+
+        let container = loadGetStartedView()
+        container.snp_makeConstraints { make in
+            make.width.equalTo(self.view)
+            make.centerY.equalTo(self.view)
+        }
+    }
+
+    private func loadGetStartedView() -> UIView {
+        let container = UIView()
+        self.view.addSubview(container)
+
+        icon = UIImageView()
+        icon.image = UIImage(named: "fxa_get_started.png")!
+        container.addSubview(icon)
+
+        error = UILabel()
+        error.textColor = UIColor(red: 214/255.0, green: 57/255.0, blue: 32/255.0, alpha: 1.0)
+        error.textAlignment = NSTextAlignment.Center
```

Comment:
You probably don't need NSTextAlignment here. i.e. error.textAlignment = .Center